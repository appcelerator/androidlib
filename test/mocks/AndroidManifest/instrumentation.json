{
	"instrumentation": [
		{
			"android:label": "Local Sample",
			"android:name": ".app.LocalSampleInstrumentation",
			"android:targetPackage": "com.example.android.apis"
		},
		{
			"android:functionalTest": true,
			"android:handleProfiling": true,
			"android:icon": "drawable resource",
			"android:label": "Local Test",
			"android:name": ".app.LocalTestInstrumentation",
			"android:targetPackage": "com.example.test.apis"
		}
	]
}
