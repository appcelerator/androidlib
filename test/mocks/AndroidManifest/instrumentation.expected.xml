<?xml version="1.0" encoding="utf-8"?>
<manifest>
  <instrumentation android:label="Local Sample" android:name=".app.LocalSampleInstrumentation" android:targetPackage="com.example.android.apis"/>
  <instrumentation android:functionalTest="true" android:handleProfiling="true" android:icon="drawable resource" android:label="Local Test" android:name=".app.LocalTestInstrumentation" android:targetPackage="com.example.test.apis"/>
</manifest>
